Guide/Tutorials/CreatingLanguages.md

```diff
@@ -48,11 +48,17 @@ There are some extendables to help use languages. msg.language is a read-only pr
 // Will skip the configs fetch and return the language which name is that one.
 this.client.languages.get('en-US');
 
-// returns 'Default Language' in this example, but if the guild has fr-FR as the configured language it would respond 'Langue par défaut'
+/*
+returns 'Default Language' in this example,
```

Comment:
if using /**/ format as you would a jsdoc block
```js
/*
 *
 *
 */
```

Reply:
Yeah was about todo that 👍